mod-di-converter-storage-server/src/main/resources/templates/db_scripts/associations-migration-gen-table/actualize_schema_for_migrations.sql

```diff
@@ -5,5 +5,4 @@ CREATE OR REPLACE FUNCTION  get_profile_association_snapshot(_master_table text,
 CREATE OR REPLACE RULE delete_associations_with_details AS  ON DELETE TO associations_view  DO INSTEAD (    DELETE FROM profile_associations WHERE OLD.master_id IN      (SELECT action_profile_id FROM profile_wrappers WHERE id = profile_associations.master_wrapper_id);    DELETE FROM profile_associations WHERE OLD.master_id IN      (SELECT job_profile_id FROM profile_wrappers WHERE id = profile_associations.master_wrapper_id);    DELETE FROM profile_associations WHERE OLD.master_id IN      (SELECT match_profile_id FROM profile_wrappers WHERE id = profile_associations.master_wrapper_id);    );
 
-CREATE OR REPLACE FUNCTION get_profile_snapshot(profileId uuid, profile_type text, profile_table text, jobProfileId text)RETURNS TABLE(snapshot json)  AS $$    BEGIN      RETURN query        EXECUTE format('          WITH RECURSIVE recursive_snapshot AS (              SELECT                job_profile.id AS association_id,                CAST(NULL AS uuid) AS master_id,                job_profile.id AS detail_id,        CAST(NULL AS uuid) AS masterwrapperid,                pw.id AS detailwrapperid,                ''%s'' detail_type,                null AS master_type,                0 AS detail_order,                json_agg(job_profile.jsonb) detail,                null AS react_to              FROM %s AS job_profile LEFT JOIN profile_wrappers pw ON pw.job_profile_id = job_profile.id              WHERE job_profile.id = ''%s''              GROUP BY job_profile.id, pw.id                UNION ALL              SELECT                associations_view.association_id,                associations_view.master_id AS master_id,                associations_view.detail_id AS detail_id,                associations_view.masterwrapperid AS masterwrapperid,                associations_view.detailwrapperid AS detailwrapperid,                associations_view.detail_type AS detail_type,                associations_view.master_type AS master_type,                associations_view.detail_order AS detail_order,                associations_view.detail AS detail,                associations_view.react_to AS react_to              FROM associations_view INNER JOIN recursive_snapshot ON associations_view.masterwrapperid = recursive_snapshot.detailwrapperid                AND CASE WHEN associations_view.master_type = ''MATCH_PROFILE'' AND ''%s'' != ''null'' THEN associations_view.job_profile_id = NULLIF(''%s'',''null'')::uuid                         ELSE associations_view.job_profile_id IS NULL END)          SELECT row_to_json(row) FROM recursive_snapshot row ORDER BY row.detail_order ASC',          profile_type, profile_table, profileId, jobProfileId, jobProfileId);    END $$language plpgsql;
```

Comment:
@RomanChernetskyi  Why this script was removed?

Reply:
@VRohach, because this script executes at mod-di-converter-storage-server/src/main/resources/templates/db_scripts/get_profile_snapshot.sql

Reply:
@RomanChernetskyi  Thank you! So this is a duplicate.